### ISSUE TITLE
Meta: When should Fetch integration happen?

### ISSUE THREAD

**Issue by yoavweiss** [w3c/resource-timing#179]
@annevk has [commented](https://github.com/w3c/resource-timing/issues/178#issuecomment-442352055) on #178, saying that we should not postpone work that depends on Fetch integration, but rather tackle it immediately.

I think that we all recognize we have a lot of tech debt to pay here when it comes to Fetch integration. To that end, we have 2 options:
1) Hold off L2 until all Fetch integration is complete. That's likely to delay publication by a long while, as we would add tests which current implementations are likely to fail due to current incompatibilities. That would also mean that L3 related features will be held off until all that work is complete.
2) Publish L2 ASAP while resolving all tech debt unrelated to Fetch integration, and start work on L3 and Fetch integration immediately after that. At that point, L3 feature work and the merge of the spec with the NavigationTiming spec can happen in parallel.

I believe 2) is the shortest path to pay off our debt here, while enabling us to ship L2, which is long overdue.

**Comment by igrigorik** [comment]
Based on previous group discussions, we agreed on (2) and reflected this in [our new charter](https://www.w3.org/2018/07/webperf-charter) — see note on Nav Timing. Our goal is to address the last batch of outstanding L2 questions, freeze and lock the L2, and then immediately start work on L3 where we integrate RT+NT on top of Fetch. 

Closing, since I believe we made an explicit decision on this one already, but if there is more to discuss here, please feel free to reopen.
